lighty-examples/lighty-community-aaa-restconf-app/src/main/java/io/lighty/kit/examples/community/aaa/restconf/Main.java

```diff
@@ -108,15 +109,20 @@ private static void startLighty(final ControllerConfiguration controllerConfigur
 
         communityRestConf.startServer();
 
-        Runtime.getRuntime().addShutdownHook(new ShutdownHook(lightyController, communityRestConf, aaaLighty));
+        //3. Register shutdown hook for graceful shutdown.
+        shutdownHook = new ShutdownHook(lightyController, communityRestConf, aaaLighty);
+        if (registerShutdownHook) {
+            Runtime.getRuntime().addShutdownHook(shutdownHook);
+        }
     }
 
-    public static void start() throws Exception {
-        main(new String[]{});
+    public void shutdown() {
+        shutdownHook.execute();
     }
 
-    private Main() {
-        /** Hide constructor, this is a utility class **/
+    public static void main(String[] args) {
```

Comment:
Move main method to the top of this class

Reply:
done